packages/visx-zoom/src/Zoom.tsx

```diff
@@ -79,229 +80,267 @@ type ZoomState = {
   isDragging: boolean;
 };
 
-class Zoom extends React.Component<ZoomProps, ZoomState> {
-  static defaultProps = {
-    passive: false,
-    scaleXMin: 0,
-    scaleXMax: Infinity,
-    scaleYMin: 0,
-    scaleYMax: Infinity,
-    transformMatrix: {
-      scaleX: 1,
-      scaleY: 1,
-      translateX: 0,
-      translateY: 0,
-      skewX: 0,
-      skewY: 0,
-    },
-    wheelDelta: (event: React.WheelEvent | WheelEvent) =>
-      -event.deltaY > 0 ? { scaleX: 1.1, scaleY: 1.1 } : { scaleX: 0.9, scaleY: 0.9 },
-    style: undefined,
-    className: undefined,
-  };
-
-  containerRef: HTMLDivElement | null = null;
-
-  startPoint: Point | undefined = undefined;
+function Zoom<ElementType extends Element>({
+  scaleXMin = 0,
+  scaleXMax = Infinity,
+  scaleYMin = 0,
+  scaleYMax = Infinity,
+  initialTransformMatrix = {
+    scaleX: 1,
+    scaleY: 1,
+    translateX: 0,
+    translateY: 0,
+    skewX: 0,
+    skewY: 0,
+  },
+  wheelDelta = (event: React.WheelEvent | WheelEvent) =>
+    -event.deltaY > 0 ? { scaleX: 1.1, scaleY: 1.1 } : { scaleX: 0.9, scaleY: 0.9 },
+  pinchDelta = ({ offset: [s], lastOffset: [lastS] }) => ({
```

Comment:
ditto on pulling out this as a constant, sorry I didn't catch these earlier. works okay as a class component defaultProps but less well for a function component.